HOWTO.md

```diff
@@ -521,10 +542,14 @@ Please mind how the configuration file must be specifically named (e.g. `-` VS
    Files/manufacturing_server.yml](#manufacturing_serveryml) and place it either in
    `/usr/share/fdo`, `/etc/fdo/` or
    `/etc/fdo/manufacturing-serverr.conf.d/`. The paths will be checked in that
-   same order.
-
-3. Execute `fdo-manufacturing-server` or run it as a service, see sample
-   file in
+   same order. 
+   Mention the path for keys and certificates as generated in step1 inside `manufcturing-server.yml`.
```

Comment:
This second line is not needed, the reader already knows how to generate keys if they have been reading the steps.